crates/tabby-common/src/path.rs

```diff
@@ -6,7 +6,7 @@ lazy_static! {
     static ref TABBY_ROOT: Mutex<Cell<PathBuf>> = {
         Mutex::new(Cell::new(match env::var("TABBY_ROOT") {
             Ok(x) => PathBuf::from(x),
-            Err(_) => PathBuf::from(env::var("HOME").unwrap()).join(".tabby"),
+            Err(_) => home::home_dir().unwrap().join(".tabby"),
```

Comment:
On Windows platform, there is no default `HOME` env, by right it should be `USERPROFILE` or `HOMEDRIVE+HOMEPATH`.

But let's borrow help from a lib called `home` which is a small used by `Cargo` & `rustup`, so should be working for most of platforms.